docs/security/readme.md

```diff
@@ -0,0 +1,6 @@
+# Mitigations for known Security vulnerabilites
+
+# CosmosDB Jupyter Notebook vulnerability
+The details of this vulnerability are described in detail [here](https://msrc-blog.microsoft.com/2021/08/27/update-on-vulnerability-in-the-azure-cosmos-db-jupyter-notebook-feature/).
+The steps to mitigate it in the Industrial-IoT Platform are described [here](cosmosdb-jupyter-notebook-2021-08-12.md)
```

Comment:
The "details are described in detail" is too much detail :).
Also, please avoid creating links with the label "here" as suggested by the [W3C](https://www.w3.org/QA/Tips/noClickHere). Here's an alternative:
See the [Update on the vulnerability in the Azure Cosmos DB Jupyter Notebook Feature](https://msrc-blog.microsoft.com/2021/08/27/update-on-vulnerability-in-the-azure-cosmos-db-jupyter-notebook-feature/) for more details.
Here are [steps to mitigate](cosmosdb-jupyter-notebook-2021-08-12.md) it in the Industrial-IoT Platform.

Reply:
thx